### PR TITLE
Link broken Fixed

### LINKED ISSUE #133: Links not working Naruto Shippuden OST - Ashura-Indra
#### This issue is about {Naruto Shippuden} (OST).

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
- [x] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [x] Assign this issue to me.

### PR DESCRIPTION
**Pull request by DivyeshPuri**
Fixes #133 

#### This PR is for [Naruto Shippuden].

- [x] This issue is for enhancement/bug fixing.
- [] This issue is an addition to JSON database.
	- [] Pull Request Title is of form `Added [Anime Name]`.
- [x] This addition is already on ongaku.
	- [x] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [x] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
@Lucifer24 change your PR comment to : 

Fixes #133

#### This PR is for Naruto Shippuden.

- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
   - [ ] Pull Request Title is of form Added [Anime Name].
- [x] This addition is already on ongaku.
- [x] This PR is an improvement to current additions.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through Contribution Guidelines.
- [x] This pull request will be closed if I failed to update it even once in a week.

i.e. a PR shouldn't be both for enhancement/bug fixing and an addition to JSON database.
and also checkboxes properly either they should be empyt like - [ ] or like - [x]

**Review by aashutoshrathi — APPROVED** [review]

File: js/data.json [hunk 1]
```diff
@@ -827,7 +827,7 @@ osts = [
 	},
 
 	{
-		"link" : "http://66.90.93.122/ost/naruto-shippuden-original-soundtrack-3/ygmuwqzvwj/08.-ashura-indra.mp3",
+		"link" : "https://www.youtubeinmp3.com/download/get/?i=9kEnefIYeIqtlqU4BlSYsPcD3lWoh3SZ&e=92&v=8-tyjXQn-Cw&progressType=button&color=008000",
```
Comment (aashutoshrathi) [inline js/data.json]: This link never returns a mp3 file, __For links of YoutubeinMP3 to return mp3 files, change your download to fetch__: 
For eg. here change your link to 
`https://www.youtubeinmp3.com/fetch/?video=https://www.youtube.com/watch?v=8-tyjXQn-Cw`
and also see that your current link never return a valid mp3, so you also shouldn't tick : 
- [ ] I have checked my addition locally and it is working and won't break existing entries.
###### PS - There may be some intentions of mine for writing large comments.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Please make changes

**Comment by DivyeshPuri** [comment]
@aashutoshrathi i think its fine now

**Comment by anshumanv** [comment]
The current link returns proper mp3 response, please cross-check.

**Review by aashutoshrathi — COMMENTED** [review]
:+1: Simply Perfect 
##### Can't write much here

**Comment by anshumanv** [comment]
Closing, as current link works perfectly fine.

**Pull request closed without merging**